src/board/mod.rs

```diff
@@ -579,32 +577,16 @@ impl Board {
             .count()
     }
     
-    pub fn new_chain_liberties(&self, m: Move) -> usize {
```

Comment:
This method and the one below definitely don't belong into Board. But it seems that by removing them we now test less than before. Do you think it would be possible to move them into the test module?